openapi/views/base.py

```diff
@@ -51,7 +51,7 @@ def has_perms(self, request_perms):
     def get_perms(self):
         self.user_perms = []
         if self.is_enterprise_admin:
-            self.user_perms = get_enterprise_adminer_codes()
+            self.user_perms = list(get_enterprise_adminer_codes())
```

Comment:
why not function get_enterprise_adminer_codes return list ?

Reply:
Codes are unique, and set elements are unique